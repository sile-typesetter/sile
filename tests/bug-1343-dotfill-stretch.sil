\begin[papersize=a5]{document}
\nofolios
\script[src=packages/leaders]
\script[src=packages/lorem]
\lorem[words=15]\par% for visual comparison dotfill vs. justified text line
|\hfill|\par% for visual comparison dotfill vs. hfill
|\dotfill|\par
|\dotfill\hbox{}\par
|Lorem\dotfill{}ipsum dolor|\par
\end{document}
