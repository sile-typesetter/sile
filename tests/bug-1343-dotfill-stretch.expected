Set paper size 	419.5275636	595.275597
Begin page
Mx 	40.9764
My 	37.3321
Set font 	Gentium Plus;10;400;;normal;;LTR
T	79 82 85 72 80 w=24.3506	(lorem)
Mx 	68.0973
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	96.0678
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	120.7668
T	86 76 87 w=10.0146	(sit)
Mx 	133.5518
T	68 80 72 87 w=20.6836	(amet)
Mx 	157.0057
T	70 82 81 86 72 87 72 87 88 85 w=44.1602	(consetetur)
Mx 	203.9361
T	86 68 71 76 83 86 70 76 81 74 w=42.9980	(sadipscing)
Mx 	249.7045
T	72 79 76 87 85 w=17.4414	(elitr)
Mx 	269.9162
T	86 72 71 w=13.6816	(sed)
Mx 	286.3682
T	71 76 68 80 w=20.5322	(diam)
Mx 	309.6707
T	81 82 81 88 80 92 w=34.3262	(nonumy)
Mx 	346.7672
T	72 76 85 80 82 71 w=29.5508	(eirmod)
Mx 	379.0883
T	87 72 80 w=16.0938	(tem)
Mx 	395.1820
T	16 w=3.3691	(-)
Mx 	20.9764
My 	49.3321
T	83 82 85 w=14.2871	(por)
Mx 	37.9406
T	76 81 89 76 71 88 81 87 w=35.3271	(invidunt)
Mx 	75.9449
T	88 87 w=8.7402	(ut)
Mx 	40.9764
My 	61.3321
T	95 w=2.4414	(|)
Mx 	396.1098
T	95 w=2.4414	(|)
Mx 	40.9764
My 	73.3321
T	95 w=2.4414	(|)
Mx 	48.6293
Mx 	51.1293
T	17 w=2.2900	(.)
Mx 	55.9193
Mx 	58.4193
T	17 w=2.2900	(.)
Mx 	63.2094
Mx 	65.7094
T	17 w=2.2900	(.)
Mx 	70.4994
Mx 	72.9994
T	17 w=2.2900	(.)
Mx 	77.7895
Mx 	80.2895
T	17 w=2.2900	(.)
Mx 	85.0795
Mx 	87.5795
T	17 w=2.2900	(.)
Mx 	92.3695
Mx 	94.8695
T	17 w=2.2900	(.)
Mx 	99.6596
Mx 	102.1596
T	17 w=2.2900	(.)
Mx 	106.9496
Mx 	109.4496
T	17 w=2.2900	(.)
Mx 	114.2397
Mx 	116.7397
T	17 w=2.2900	(.)
Mx 	121.5297
Mx 	124.0297
T	17 w=2.2900	(.)
Mx 	128.8197
Mx 	131.3197
T	17 w=2.2900	(.)
Mx 	136.1098
Mx 	138.6098
T	17 w=2.2900	(.)
Mx 	143.3998
Mx 	145.8998
T	17 w=2.2900	(.)
Mx 	150.6899
Mx 	153.1899
T	17 w=2.2900	(.)
Mx 	157.9799
Mx 	160.4799
T	17 w=2.2900	(.)
Mx 	165.2699
Mx 	167.7699
T	17 w=2.2900	(.)
Mx 	172.5600
Mx 	175.0600
T	17 w=2.2900	(.)
Mx 	179.8500
Mx 	182.3500
T	17 w=2.2900	(.)
Mx 	187.1401
Mx 	189.6401
T	17 w=2.2900	(.)
Mx 	194.4301
Mx 	196.9301
T	17 w=2.2900	(.)
Mx 	201.7201
Mx 	204.2201
T	17 w=2.2900	(.)
Mx 	209.0102
Mx 	211.5102
T	17 w=2.2900	(.)
Mx 	216.3002
Mx 	218.8002
T	17 w=2.2900	(.)
Mx 	223.5902
Mx 	226.0902
T	17 w=2.2900	(.)
Mx 	230.8803
Mx 	233.3803
T	17 w=2.2900	(.)
Mx 	238.1703
Mx 	240.6703
T	17 w=2.2900	(.)
Mx 	245.4604
Mx 	247.9604
T	17 w=2.2900	(.)
Mx 	252.7504
Mx 	255.2504
T	17 w=2.2900	(.)
Mx 	260.0404
Mx 	262.5404
T	17 w=2.2900	(.)
Mx 	267.3305
Mx 	269.8305
T	17 w=2.2900	(.)
Mx 	274.6205
Mx 	277.1205
T	17 w=2.2900	(.)
Mx 	281.9106
Mx 	284.4106
T	17 w=2.2900	(.)
Mx 	289.2006
Mx 	291.7006
T	17 w=2.2900	(.)
Mx 	296.4906
Mx 	298.9906
T	17 w=2.2900	(.)
Mx 	303.7807
Mx 	306.2807
T	17 w=2.2900	(.)
Mx 	311.0707
Mx 	313.5707
T	17 w=2.2900	(.)
Mx 	318.3608
Mx 	320.8608
T	17 w=2.2900	(.)
Mx 	325.6508
Mx 	328.1508
T	17 w=2.2900	(.)
Mx 	332.9408
Mx 	335.4408
T	17 w=2.2900	(.)
Mx 	340.2309
Mx 	342.7309
T	17 w=2.2900	(.)
Mx 	347.5209
Mx 	350.0209
T	17 w=2.2900	(.)
Mx 	354.8110
Mx 	357.3110
T	17 w=2.2900	(.)
Mx 	362.1010
Mx 	364.6010
T	17 w=2.2900	(.)
Mx 	369.3910
Mx 	371.8910
T	17 w=2.2900	(.)
Mx 	376.6811
Mx 	379.1811
T	17 w=2.2900	(.)
Mx 	383.9711
Mx 	386.4711
T	17 w=2.2900	(.)
Mx 	396.1098
T	95 w=2.4414	(|)
Mx 	40.9764
My 	85.3321
T	95 w=2.4414	(|)
Mx 	48.6293
Mx 	51.1293
T	17 w=2.2900	(.)
Mx 	55.9193
Mx 	58.4193
T	17 w=2.2900	(.)
Mx 	63.2094
Mx 	65.7094
T	17 w=2.2900	(.)
Mx 	70.4994
Mx 	72.9994
T	17 w=2.2900	(.)
Mx 	77.7895
Mx 	80.2895
T	17 w=2.2900	(.)
Mx 	85.0795
Mx 	87.5795
T	17 w=2.2900	(.)
Mx 	92.3695
Mx 	94.8695
T	17 w=2.2900	(.)
Mx 	99.6596
Mx 	102.1596
T	17 w=2.2900	(.)
Mx 	106.9496
Mx 	109.4496
T	17 w=2.2900	(.)
Mx 	114.2397
Mx 	116.7397
T	17 w=2.2900	(.)
Mx 	121.5297
Mx 	124.0297
T	17 w=2.2900	(.)
Mx 	128.8197
Mx 	131.3197
T	17 w=2.2900	(.)
Mx 	136.1098
Mx 	138.6098
T	17 w=2.2900	(.)
Mx 	143.3998
Mx 	145.8998
T	17 w=2.2900	(.)
Mx 	150.6899
Mx 	153.1899
T	17 w=2.2900	(.)
Mx 	157.9799
Mx 	160.4799
T	17 w=2.2900	(.)
Mx 	165.2699
Mx 	167.7699
T	17 w=2.2900	(.)
Mx 	172.5600
Mx 	175.0600
T	17 w=2.2900	(.)
Mx 	179.8500
Mx 	182.3500
T	17 w=2.2900	(.)
Mx 	187.1401
Mx 	189.6401
T	17 w=2.2900	(.)
Mx 	194.4301
Mx 	196.9301
T	17 w=2.2900	(.)
Mx 	201.7201
Mx 	204.2201
T	17 w=2.2900	(.)
Mx 	209.0102
Mx 	211.5102
T	17 w=2.2900	(.)
Mx 	216.3002
Mx 	218.8002
T	17 w=2.2900	(.)
Mx 	223.5902
Mx 	226.0902
T	17 w=2.2900	(.)
Mx 	230.8803
Mx 	233.3803
T	17 w=2.2900	(.)
Mx 	238.1703
Mx 	240.6703
T	17 w=2.2900	(.)
Mx 	245.4604
Mx 	247.9604
T	17 w=2.2900	(.)
Mx 	252.7504
Mx 	255.2504
T	17 w=2.2900	(.)
Mx 	260.0404
Mx 	262.5404
T	17 w=2.2900	(.)
Mx 	267.3305
Mx 	269.8305
T	17 w=2.2900	(.)
Mx 	274.6205
Mx 	277.1205
T	17 w=2.2900	(.)
Mx 	281.9106
Mx 	284.4106
T	17 w=2.2900	(.)
Mx 	289.2006
Mx 	291.7006
T	17 w=2.2900	(.)
Mx 	296.4906
Mx 	298.9906
T	17 w=2.2900	(.)
Mx 	303.7807
Mx 	306.2807
T	17 w=2.2900	(.)
Mx 	311.0707
Mx 	313.5707
T	17 w=2.2900	(.)
Mx 	318.3608
Mx 	320.8608
T	17 w=2.2900	(.)
Mx 	325.6508
Mx 	328.1508
T	17 w=2.2900	(.)
Mx 	332.9408
Mx 	335.4408
T	17 w=2.2900	(.)
Mx 	340.2309
Mx 	342.7309
T	17 w=2.2900	(.)
Mx 	347.5209
Mx 	350.0209
T	17 w=2.2900	(.)
Mx 	354.8110
Mx 	357.3110
T	17 w=2.2900	(.)
Mx 	362.1010
Mx 	364.6010
T	17 w=2.2900	(.)
Mx 	369.3910
Mx 	371.8910
T	17 w=2.2900	(.)
Mx 	376.6811
Mx 	379.1811
T	17 w=2.2900	(.)
Mx 	383.9711
Mx 	386.4711
T	17 w=2.2900	(.)
Mx 	391.2611
Mx 	393.7611
T	17 w=2.2900	(.)
Mx 	398.5512
Mx 	40.9764
My 	97.3321
T	95 w=2.4414	(|)
Mx 	43.4178
T	47 82 85 72 80 w=26.4014	(Lorem)
Mx 	70.4994
Mx 	72.9994
T	17 w=2.2900	(.)
Mx 	77.7895
Mx 	80.2895
T	17 w=2.2900	(.)
Mx 	85.0795
Mx 	87.5795
T	17 w=2.2900	(.)
Mx 	92.3695
Mx 	94.8695
T	17 w=2.2900	(.)
Mx 	99.6596
Mx 	102.1596
T	17 w=2.2900	(.)
Mx 	106.9496
Mx 	109.4496
T	17 w=2.2900	(.)
Mx 	114.2397
Mx 	116.7397
T	17 w=2.2900	(.)
Mx 	121.5297
Mx 	124.0297
T	17 w=2.2900	(.)
Mx 	128.8197
Mx 	131.3197
T	17 w=2.2900	(.)
Mx 	136.1098
Mx 	138.6098
T	17 w=2.2900	(.)
Mx 	143.3998
Mx 	145.8998
T	17 w=2.2900	(.)
Mx 	150.6899
Mx 	153.1899
T	17 w=2.2900	(.)
Mx 	157.9799
Mx 	160.4799
T	17 w=2.2900	(.)
Mx 	165.2699
Mx 	167.7699
T	17 w=2.2900	(.)
Mx 	172.5600
Mx 	175.0600
T	17 w=2.2900	(.)
Mx 	179.8500
Mx 	182.3500
T	17 w=2.2900	(.)
Mx 	187.1401
Mx 	189.6401
T	17 w=2.2900	(.)
Mx 	194.4301
Mx 	196.9301
T	17 w=2.2900	(.)
Mx 	201.7201
Mx 	204.2201
T	17 w=2.2900	(.)
Mx 	209.0102
Mx 	211.5102
T	17 w=2.2900	(.)
Mx 	216.3002
Mx 	218.8002
T	17 w=2.2900	(.)
Mx 	223.5902
Mx 	226.0902
T	17 w=2.2900	(.)
Mx 	230.8803
Mx 	233.3803
T	17 w=2.2900	(.)
Mx 	238.1703
Mx 	240.6703
T	17 w=2.2900	(.)
Mx 	245.4604
Mx 	247.9604
T	17 w=2.2900	(.)
Mx 	252.7504
Mx 	255.2504
T	17 w=2.2900	(.)
Mx 	260.0404
Mx 	262.5404
T	17 w=2.2900	(.)
Mx 	267.3305
Mx 	269.8305
T	17 w=2.2900	(.)
Mx 	274.6205
Mx 	277.1205
T	17 w=2.2900	(.)
Mx 	281.9106
Mx 	284.4106
T	17 w=2.2900	(.)
Mx 	289.2006
Mx 	291.7006
T	17 w=2.2900	(.)
Mx 	296.4906
Mx 	298.9906
T	17 w=2.2900	(.)
Mx 	303.7807
Mx 	306.2807
T	17 w=2.2900	(.)
Mx 	311.0707
Mx 	313.5707
T	17 w=2.2900	(.)
Mx 	318.3608
Mx 	320.8608
T	17 w=2.2900	(.)
Mx 	325.6508
Mx 	328.1508
T	17 w=2.2900	(.)
Mx 	332.9408
Mx 	335.4408
T	17 w=2.2900	(.)
Mx 	346.3383
T	76 83 86 88 80 w=25.2002	(ipsum)
Mx 	374.1811
T	71 82 79 82 85 w=21.9287	(dolor)
Mx 	396.1098
T	95 w=2.4414	(|)
End page
Finish
